src/components/Api.js

```diff
@@ -0,0 +1,185 @@
+export default class Api {
+  constructor(options) {
+    // constructor body
+    this.options = options;
+  }
+
+  getInitialCards() {
+    return fetch("https://around.nomoreparties.co/v1/cohort-3-en/cards/", {
+      method: "GET",
+      headers: {
+        authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+    });
+  }
+
+  getUserInfo() {
+    return fetch("https://around.nomoreparties.co/v1/cohort-3-en/users/me", {
+      method: "GET",
+      headers: {
+        authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+    });
+  }
+
+  editUserInfo(values) {
+    return fetch("https://around.nomoreparties.co/v1/cohort-3-en/users/me", {
+      method: "PATCH",
+      headers: {
+        authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        name: values.name,
+        about: values.subtitle,
+      }),
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject(`Error: ${res.status}`);
+      })
+      .catch((err) => {
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very end of any request and this end is in `index.js` after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.